### PR TITLE
cli release 1.4

### PR DESCRIPTION
**Pull request by sunkickr**
Astro release notes for CLI 1.4.0 release

Resolves https://github.com/astronomer/docs/issues/1017

### CODE REVIEW

**Comment by jwitz** [comment]
Thanks for writing these up @sunkickr, I jumped in and made some changes! I still need to come back to this and update places where we tell users to manually `docker exec`, but wanted to share this draft now so you can check it for technical accuracy.

File: astro/cli/astro-deployment-variable-create.md [hunk 1]
```diff
@@ -32,23 +30,21 @@ After setting the variables, this command works for a Deployment and you don't n
...
 
 ## Examples
 
 ```sh
 # Create a new secret environment variable
-$ astro deployment variable create --deployment-id cl03oiq7d80402nwn7fsl3dmv --key AIRFLOW__SECRETS__BACKEND_KWARGS --value <my-secret-value> --secret
+$ astro deployment variable create --deployment-id cl03oiq7d80402nwn7fsl3dmv AIRFLOW__SECRETS__BACKEND_KWARGS=<my-secret-value> --secret
+
+# Create multiple environment variables for a Deployment at once by specifying multiple keys
+$ astro deployment variable create --deployment-id cl03oiq7d80402nwn7fsl3dmv AIRFLOW__CORE__PARALLELISM=32 MAX_ACTIVE_TASKS_PER_DAG=16
```
Comment (sunkickr) [inline astro/cli/astro-deployment-variable-create.md:44]:
```suggestion
$ astro deployment variable create AIRFLOW__CORE__PARALLELISM=32 MAX_ACTIVE_TASKS_PER_DAG=16 --deployment-id cl03oiq7d80402nwn7fsl3dmv
```

Comment (sunkickr) [reply astro/cli/astro-deployment-variable-create.md:44]: I'm not sure that the other way will work

File: astro/cli/astro-user-invite.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+---
+sidebar_label: 'astro user invite'
+title: 'astro user invite'
+id: astro-user-invite
+description: Reference documentation for astro user invite.
+---
+
+Invite a new user to the Organization you're currently authenticated to. You must be have Organization Owner to run this command.
```
Comment (sunkickr) [inline astro/cli/astro-user-invite.md:8]:
```suggestion
Invite a new user to the Organization you're currently authenticated to. You must have Organization Owner to run this command.  
```

File: astro/cli/astro-user-invite.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+| Option              | Description                                                                                   | Possible Values                                 |
+| ------------------- | --------------------------------------------------------------------------------------------- | ----------------------------------------------- |
+| `--r`, `--role` | Force the user to manually access the Cloud UI to log in instead of opening the browser automatically from the CLI.           | `ORGANIZATION_MEMBER`, `ORGANIZATION_BILLING_ADMIN`, `ORGANIZATION_OWNER`|
+
+## Examples
+
+```sh
+$ astro user invite
+# CLI prompts you for a user's email and assigns them the `ORGANIZATION_MEMBER` role by default
+$ astro user invite user@company.com ORGANIZATION_BILLING_ADMIN
```
Comment (sunkickr) [inline astro/cli/astro-user-invite.md:27]:
```suggestion
$ astro user invite user@company.com --role ORGANIZATION_BILLING_ADMIN
```

**Review by sunkickr — COMMENTED** [review]
looks good. Just have a few suggestions

File: astro/cli/astro-dev-bash.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+astro dev bash
+```
+
+By default, the command execs into the scheduler container and prompts you to run a bash command. To run a command in a different container, you have to specify a different container flag.
+
+## Options
+
+| Option              | Description                                           | Possible Values |
+| ------------------- | ----------------------------------------------------- | --------------- |
+| `-p`, `postgres`    | Run a bash command in the metadata database container | ``              |
```
Comment (jwitz) [inline astro/cli/astro-dev-bash.md:24]:
```suggestion
| `-p`, `--postgres`    | Run a bash command in the metadata database container | ``              |
```

File: astro/cli/astro-user-invite.md [hunk 5]
```diff
@@ -0,0 +1,33 @@
+---
+sidebar_label: 'astro user invite'
+title: 'astro user invite'
+id: astro-user-invite
+description: Reference documentation for astro user invite.
+---
+
+Invite a new user to the Organization you're currently authenticated to. You must have Organization Owner to run this command.
```
Comment (jwitz) [inline astro/cli/astro-user-invite.md:8]:
```suggestion
Invite a new user to the Organization you're currently authenticated to. You must have Organization Owner permissions to run this command.  
```

File: astro/develop-project.md [hunk 6]
```diff
@@ -167,14 +163,16 @@ To build additional helper functions for DAGs into your Astro project, Astronome
...
 2. [Restart your local environment](develop-project.md#restart-your-local-environment).
 
-To confirm that your helper functions were successfully installed:
+To confirm that your helper functions were successfully installed, run the following command:
 
-1. Run `docker ps` to identify the 3 running docker containers on your machine
-2. Copy the container ID of your scheduler container
-3. Run the following command to see your new directory in the container:
+    ```sh
+    astro dev bash --scheduler "/bin/bash"
```
Comment (jwitz) [inline astro/develop-project.md:169]: @sunkickr Can you confirm that this is the right format for this command?

File: astro/develop-project.md [hunk 7]
```diff
@@ -261,10 +259,10 @@ services:
 
 Make sure to specify `version: "3.1"` and follow the format of the source code file linked above.
 
-To see your override file live in your local Airflow environment, run the following command for any container running Airflow:
+To see your override file live in your local Airflow environment, run the following command to see the file in your scheduler container:
 
 ```sh
-docker exec -it <container-name> ls -al
+astro dev bash --scheduler ls -al
```
Comment (jwitz) [inline astro/develop-project.md:265]:
```suggestion
astro dev bash --scheduler "ls -al"
```

**Review by jwitz — APPROVED** [review]

**Review by paolaperaza — APPROVED** [review]

**Pull request merged**